daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -599,6 +599,7 @@ func (devices *DeviceSet) cleanupDeletedDevices() error {
 
 	// If there are no deleted devices, there is nothing to do.
 	if devices.nrDeletedDevices == 0 {
+		devices.Unlock()
```

Comment:
let's do `defer`, so next time when someone decide to fix this function won't forget to add unlock before return.